src/components/Footer/Footer.module.scss

```diff
@@ -1 +1,87 @@
+.outer {
+  --color-bg-grey: #d4d4d4;
+  --color-black: #171717;
+  --color-grey: #f5f5f5;
 
+  color: var(--color-black);
+  background: var(--color-bg-grey);
+  border-top: 2px solid var(--color-black);
+  padding: 32px 20px;
+}
+
+.inner {
+  display: flex;
+  justify-content: space-between;
+  word-break: break-word;
+  width: 100%;
+}
+
+.content {
+  display: none;
+  @media (min-width: 768px) {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 20px;
+  }
+}
+
+.category {
+  width: 144px;
+}
+
+.categoryLabel {
+  font-size: 1em;
+  line-height: 1.2em;
+  font-weight: 600;
+}
+
+.categoryList {
+  padding: 0;
+  list-style: none;
+  margin-top: 12px;
+}
+
+.categoryItem {
+  margin-bottom: 8px;
+}
+
+.link {
+  text-decoration: none;
+}
+
+.formContainer {
+  @media (max-width: 767px) {
```

Comment:
Nitpick but try to stick with min-width: 768px (i.e. set flex-grow normally, reset to initial value in media query)